Description: Fix build (arm-linux-gnueabi-gcc is unable to create an executable file.)
Index: ppsspp-git3/ffmpeg/linux_arm.sh
===================================================================
--- ppsspp-git3.orig/ffmpeg/linux_arm.sh	2017-04-06 21:38:09.883582891 -0300
+++ ppsspp-git3/ffmpeg/linux_arm.sh	2017-04-06 21:38:09.879582825 -0300
@@ -2,14 +2,6 @@
 
 set -e
 
-GENERAL="\
-   --enable-cross-compile \
-   --extra-libs="-lgcc" \
-   --arch=arm \
-   --cc=arm-linux-gnueabi-gcc \
-   --cross-prefix=arm-linux-gnueabi- \
-   --nm=arm-linux-gnueabi-nm"
-
 MODULES="\
    --disable-avdevice \
    --disable-filters \
@@ -101,8 +93,7 @@
 {
 ./configure --target-os=linux \
     --prefix=./linux/armv7 \
-    ${GENERAL} \
-    --extra-cflags=" -O3 -fasm -Wno-psabi -fno-short-enums -fno-strict-aliasing -finline-limit=300 -mfloat-abi=softfp -mfpu=neon -marm -march=armv7-a" \
+    --extra-cflags=" -O3 -fasm -Wno-psabi -fno-short-enums -fno-strict-aliasing -finline-limit=300 -mfloat-abi=hard -mfpu=neon -marm -march=armv7-a" \
     --disable-shared \
     --enable-static \
     --enable-zlib \
@@ -119,6 +110,5 @@
 make install
 }
 
-build_ARMv6
 build_ARMv7
 echo Linux ARM builds finished
